CMakeLists.txt

```diff
@@ -11,8 +11,16 @@ endif()
 
 include(sources.cmake)
 
+option(BUILD_TESTING "Build all test targets for AWS-LC" ON)
```

Comment:
I don't think it makes sense to have a different option. AWS-LC has a dependency on googletest, which uses BUILD_TESTING, and we would have to set this flag regardless of whether or not we create our own option of a different name. Also, I think it makes sense to use this naming convention since it is a more common practice.